mypy.ini

```diff
@@ -5,6 +5,7 @@ warn_unused_configs = True
 warn_redundant_casts = True
 show_error_codes = True
 check_untyped_defs = True
+allow_redefinition = True
```

Comment:
It sounds wrong to allow it globally, perhaps it can be limited to just one file or even better, to just one line, by using `#type: ignore[no-redef]` , as for example done here:
https://github.com/pytorch/pytorch/blob/1b55e2b043b0be0f99de20477f4135ca4e94c3d0/torch/futures/__init__.py#L16

Reply:
The issue is due to cases like this `kernel_size = _single(kernel_size)`, where int or tuple of int is converted to a tuple, which occurs at least a few times in this file. I think this pattern could be common, but I can limit it for this file for now.